Assets/Prefabs/Crossbowman.prefab

```diff
@@ -1,102 +1,5 @@
 %YAML 1.1
 %TAG !u! tag:unity3d.com,2011:
---- !u!1 &1475844385229722679
```

Comment:
@BisschenHeller You can ignore this, this only removes the cube from the crossbowmen prefab